scripts/finschia/start.sh

```diff
@@ -38,7 +37,7 @@ docker run --rm \
   --mount type=volume,source=fnsad_data,target=/root \
   "$REPOSITORY:$VERSION" \
   /template/run_finschia.sh \
-  >"$FINSCHIA_LOGFILE" 2>&1 &
+  >"$FINSCHIA_LOGFILE" >&2 &
```

Comment:
with this, no log are issued. Isn't it?

Reply:
Yes, it is. Log issued only when an error occur

Reply:
I mean I think this change is not needed and `>"$FINSCHIA_LOGFILE" 2>&1 &` works well. Sorry for confusing comment.